pkgs/development/node-packages/overrides.nix

```diff
@@ -23,6 +23,8 @@ final: prev: {
     '';
   };
 
+  "@antora/cli" = pkgs.antora;
```

Comment:
Is there already a section gated behind allowAliases?

Reply:
There is not one yet, but @lilyinstarlight mentioned the idea of making one yesterday. Might wait with merging this until it's done.

Reply:
I've got a local branch which just copied the aliases file as most other package sets did. It does mean that you can only alias within the package set though and not to an outer `pkgs`. I'll push it and open a PR so you can see it